src/network/invthread.py

```diff
@@ -42,7 +42,7 @@ def handleLocallyGenerated(stream, hashId):
         """Locally generated inventory items require special handling"""
         dandelion.instance.addHash(hashId, stream=stream)
```

Comment:
here, `addHash` could return `True` / `False` depending on whether dandelion is on/off, and we could use this for further checks.

Reply:
but for there places we have to use `dandelion.instace.enabled` in this file